WorkloadManagementSystem/JobWrapper/Watchdog.py

```diff
@@ -309,7 +309,7 @@ def _performChecks(self):
       self.parameters['RSS'].append(rss)
       msg += "Job Vsize: %.1f kb " % vsize
       msg += "Job RSS: %.1f kb " % rss
-    result = self.getDiskSpace()
+    result = self.getDiskSpace(exclude='fuse')
```

Comment:
you may want to add a comment to explain why

Reply:
Done.